docs/starters.yml

```diff
@@ -6235,3 +6235,21 @@
     - Sitemap Generation
     - XML Sitemaps
     - Progressive Web Ap
+- url: https://gatsby-starter-banshorian.now.sh
```

Comment:
You can redeploy your site on **Vercel** server by importing your repo. here _`https://vercel.com/import/git`_.

ZEIT's Now is changed to **Vercel** _`https://vercel.com/`_ and after deploying your project on Vercel server, your url will look similar to _`https://gatsby-starter-banshorian.vercel.app`_.

Thanks for raising PR.

Reply:
I just did exactly that the URL's vercel spits back at me is: 
DOMAINS
gatsby-starter-banshorian.webmaeistro.now.sh
gatsby-starter-banshorian.now.sh
gatsby-starter-banshorian-git-master.webmaeistro.now.sh

![gatsby-starter-banshorian-now-sh](https://user-images.githubusercontent.com/19165833/81866600-b5123800-956f-11ea-8c2b-7de280cf91f7.jpg)

Reply:
Okay, it seems deployment is done using Vercel CLI and Vercel is still using `.now.sh` after deployment.

Refer this `https://vercel.com` as it showcases that the output link should end with `.vercel.app`

Reply:
I also have a gatsby-starter-banshorian.netlify.app URL . I could opted for that instead to make it easy..? :D

Reply:
LGTM! Thanks